pyteal/ast/subroutine.py

```diff
@@ -299,15 +300,15 @@ class _OutputKwArgInfo:
 
     @staticmethod
     def from_dict(kwarg_info: dict[str, abi.TypeSpec]) -> Optional["_OutputKwArgInfo"]:
-        if kwarg_info is None or len(kwarg_info) == 0:
-            return None
-        elif len(kwarg_info) == 1:
-            key = list(kwarg_info.keys())[0]
-            return _OutputKwArgInfo(key, kwarg_info[key])
-        else:
-            raise TealInputError(
-                f"illegal conversion kwarg_info length {len(kwarg_info)}."
-            )
+        match list(kwarg_info.keys()):
```

Comment:
(probably) need to additionally handle `None` case, I need to remember if I really want to support that case...

Reply:
@ahangsu Let me know if you feel the `None` case needs to be handled.  As mentioned in PR description, I traced code paths and convinced myself the value won't be `None`.

However, I consider my Python interpretation abilities to be mediocre at best.  So, if you feel the `None` check is needed, I'll add it.

Reply:
okay, I took another pass, and all the use-cases for `from_dict` is on `Subroutine`'s output kwarg. It is never None, so I am happy to merge as for this one.

Reply:
@ahangsu Sounds like a plan.  Thanks for confirming - Resolving.